src/Spawner.cpp

```diff
@@ -344,6 +372,24 @@ void Spawner::waitUntilAllReady(const base::Time& timeout)
     }
 }
 
+bool Spawner::killDeployment(const std::string &dplName)
+{
+    Spawner::ProcessHandle &handle = getDeployment(dplName);
+
+    handle.sendSigInt();
+    if(!handle.wait())
```

Comment:
the parameters for wait should be configurable here. I'd suggest to add them also to killDeployment